src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import { PersonName } from './PersonName';
+import { PersonAge } from './PersonAge';
+import { PersonSpouse } from './PersonSpouse';
+
+export const Person = ({ person }) => {
+  return (
+    <section className="Person">
+      <PersonName name={person.name} />
+      {person.age && <PersonAge age={person.age} />}
+      <PersonSpouse
+        sex={person.sex}
+        spouse={person.partnerName}
+        isMarried={person.isMarried}
```

Comment:
[checklist](https://github.com/mate-academy/react_person/blob/master/checklist.md): Use destructuring for getting access to values of props object